docs/eventing/sources/ping-source/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "PingSource"
+weight: 31
+type: "docs"
+aliases:
+  - /docs/eventing/sources/pingsource
+  - ../cronjob-source
+  - /docs/eventing/samples/ping-source
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
```

Comment:
I originally intended to assign an unique color based on the maturity of the API: alpha = red, beta = orange, v1 = green. WDYT?

Reply:
I'm OK with that however I think it should be tracked somewhere which docs have which labels to make sure they're kept up to date, in general but especially if we're color coding them.

Reply:
@lionelvillard should this one be orange or green? I'm not clear on whether we want orange for beta or green for v1 for something that's called `v1beta2`

Reply:
Update to v1